nodes/mtdevice.py

```diff
@@ -577,7 +577,7 @@ def GetUTCTime(self):
             struct.unpack('!IHBBBBBB', data)
         return (ns, year, month, day, hour, minute, second, flag)
 
-    def SetUTCTime(self, ns, year, month, day, hour, minute, second, flag):
+    def SetUTCTime(self, year, month, day, hour, minute, second, ns, flag):
```

Comment:
No, don't change internal API, which is intended to follow the documentation and message. The change in order is for the command line interface only.